docs/testing-flakes.md

```diff
@@ -0,0 +1,60 @@
+# OpenShift on Azure testing flakes triaging
+
+A lot of end-to-end / conformance tests depend on resources that may vary in
+time with different workloads, software stack versions and available
+hardware. For that reason, a reasonable chunk of our test harness isn't deterministic.
+A side effect of that is that new Pull Requests advancing through the testing pipeline
+may fail with reasons that are unrelated to the changes that comprises that pull request
+but rather by some underlying components that the cluster uses to run these tests.
+Those test failures are called flakes.
+
+The following is intended as a guide for identifying those testing flakes and to
+document them as well as possible so that it's possible to come back at a later
+time and help eliminate them or lower their occurrence rate.
+
+## What to do when a PR testing fails
+
+1. Make sure the error isn't related to the change. When confirmed, it's time to investigate
+   the flake further.
+2. After initial inspection of the flake (See Tricks below), either :
+	* CREATE a new github issue to report a flake never seen before. Try to add
+          some basic info :
+		* Title starting with : "Flake:"
+		* Links to PR where encountered and to CI-operator PR diagnostic page
+                  for the failing test
+		* A brief but complete description of how it was encountered with the last
```

Comment:
move this bullet point up one.  The key thing is an appropriate snippet of log such that someone else who encounters the same flake can easily find the issue.  Additional description is helpful but not essential.  This should be in the issue description.